application/classes/Ushahidi/Console/PostExporter.php

```diff
@@ -104,21 +104,21 @@ protected function executeExport(InputInterface $input, OutputInterface $output)
 	{
 		$userContextService = service('usercontext.service');
```

Comment:
This is definitely future music... but in an ideal world all our commands would just execute a usecase with minimal setup/shutdown. Any idea how hard it would be to move most of this into a custom export usecase? how much is specific to running at the CLI? or outputting to the filesystem? Could we just set up the formatter and then handoff to a usecase?

In case its not clear: *Please don't even consider doing that work now*